Client/mods/deathmatch/logic/luadefs/CLuaVehicleDefs.cpp

```diff
@@ -4273,3 +4275,30 @@ std::variant<bool, std::array<CVector, 4>> CLuaVehicleDefs::OOP_GetVehicleEntryP
 
     return entryPoints;
 }
+
+bool CLuaVehicleDefs::AddVehicleSirens(CClientVehicle* vehicle, std::uint8_t sirenType,  std::uint8_t sirenCount, std::optional<bool> enable360, std::optional<bool> enableLOSCheck, std::optional<bool> enableRandomiser, std::optional<bool> enableSilent) noexcept
+{
+    eClientVehicleType vehicleType = vehicle->GetVehicleType();
+
+    if (vehicleType != CLIENTVEHICLE_CAR && vehicleType != CLIENTVEHICLE_MONSTERTRUCK && vehicleType != CLIENTVEHICLE_QUADBIKE &&  vehicleType != CLIENTVEHICLE_TRAIN)
```

Comment:
This list shows that only trams support sirens, so specifying train in general is a bad idea. Remove it and do it only for cars and quads as it is on the server side. This can be adjusted later in a separate PR, and at this point problems may arise due to different codes on the server and client sides

Reply:
I will try to add a model check for Tram type ID 449

Reply:
You should check whether the tram actually supports sirens or whether the list is simply incomplete